docs/tasks/access-kubernetes-api/migrate-third-party-resource.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Migrate a ThirdPartyResource to CustomResourceDefinition
+assignees:
+- enisoc
+- deads2k
+---
+
+{% capture overview %}
+This page shows how to migrate data stored in a ThirdPartyResource (TPR) to a CustomResourceDefinition (CRD).
+
+Kubernetes does not automatically migrate existing TPRs because of API changes introduced as part of
+[graduating to beta](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/thirdpartyresources.md)
+under a new name and API group.
+Instead, both TPR and CRD are available and operate independently in Kubernetes 1.7.
+
+The simplest way to migrate is to stop all clients that use a given TPR, then delete the TPR and
+start from scratch with a CRD.
+This page describes an optional process that eases the transition by migrating existing TPR data for
+you **on a best-effort basis**.
+{% endcapture %}
+
+{% capture prerequisites %}
+* Make sure your Kubernetes cluster has a **master version of exactly 1.7.x** (any patch release),
+  as this is the only version that supports both TPR and CRD.
+* If you use a TPR-based custom controller, check with the author of the controller first.
+  Some or all of these steps may be unnecessary if the custom controller handles the migration for
+  you.
+* Be familiar with the concept of [custom resources](/docs/concepts/extensions/custom-resources/),
+  which were known as *third-party resources* until Kubernetes 1.7.
+* Be familiar with [how to use CustomResourceDefinitions](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/).
+* **Before performing a migration on real data, conduct a dry run of these steps in a test cluster.**
```

Comment:
instead phrase this as "conduct a dry run by going through these steps in a test cluster"? It might be just me but for a second I thought "dry run" was a special command